base/src/org/adempiere/engine/CostEngine.java

```diff
@@ -509,11 +509,9 @@ private BigDecimal convertCostToSchemaCurrency(MAcctSchema acctSchema , IDocumen
 				model.getC_Currency_ID(), acctSchema.getC_Currency_ID() ,
 				model.getDateAcct(), model.getC_ConversionType_ID() ,
 				model.getAD_Client_ID(), model.getAD_Org_ID());
-		if (rate != null) {
+		if (rate != null) 
 			costThisLevel = cost.multiply(rate);
-			if (costThisLevel.scale() > acctSchema.getCostingPrecision())
-				costThisLevel = costThisLevel.setScale(acctSchema.getCostingPrecision(), BigDecimal.ROUND_HALF_UP);
-		}
+
```

Comment:
Why  change  the rounding standard based on the currency costing rounding or schema that use all System, and was replace for a specific rounding?